static/src/components/wizard/step.vue

```diff
@@ -64,25 +64,29 @@ export default {
           return `${this.icon} icon-default`;
       },
       activeClass: function () {
-        return this.active ? 'active-title' : '';
+        return this.active ? 'active' : '';
       }
   }
 
 };
 </script>
 
 <style>
-.stepwizard-step .active-title {
-  font-weight: bold;
-  color: #3498db;
-}
-
 .stepwizard-step {
     display: table-cell;
     text-align: center;
     position: relative;
 }
 
+.stepwizard-step.active a {
```

Comment:
Since this file is part of the wizard maybe the style can be on 'static/sass/custom/wizard.scss'

Reply:
Yes, however that might be better performed in a separate PR. This would help isolate the required testing, rather than affecting other components. Thanks.

Reply:
Yes, however that might be better performed in a separate PR. This would help isolate the required testing, rather than affecting other components. Thanks.

Reply:
Sounds good.